### PR TITLE
52 display dynamic weather icon

### LINKED ISSUE #52: Task: Dynamically display weather icon
# Parent Feature
#3 Display Local Weather

## Blockers
#51 Display Static Weather Icon
#27 Display Local Temp Dynamically

## Acceptance Criteria
Weather icon should reflect current weather at user's location.

### PR DESCRIPTION
**Pull request by jhadams82**
builds a url to the weather icon for user's current position

Closes #52

### CODE REVIEW

File: code.js [hunk 1]
```diff
@@ -40,10 +40,23 @@ function getTheWeather() {
     .then(data => {
       return data.json();
     }) //Returns a promise
-    .then(res => {
-      //Gives promise a handle (res)
+    .then(res => { //Gives promise a handle (res)
+      console.log(res);
```
Comment (MoribundMedium) [inline code.js]: Please remove `console.log()` statements before merging.

File: code.js [hunk 2]
```diff
@@ -40,10 +40,23 @@ function getTheWeather() {
...
+
+      //Display local temperature
       let fTemp = ((res.main.temp - 273.15) * 9) / 5 + 32; //Converts JSON object element res.main.temp from K° to F°
       document.querySelector("#theTemp").innerHTML = Math.round(fTemp); //Rounds and displays current temperature
+      console.log(`fTemp: ${fTemp}`);
+
+      //Display weather image
+      let iconUrl = `https://openweathermap.org/img/wn/${res.weather[0].icon}@2x.png`;
+      document.getElementById("weather-image").src = iconUrl;
+      console.log(`iconUrl: ${iconUrl}`);
```
Comment (MoribundMedium) [inline code.js]: I'd recommend moving this block of code to a helper function, then calling it from within `getTheWeather()`. At the same time, refactor L46-L48 into a helper function too. We don't want to get the logic from two different functions mixed up together.

**Review by MoribundMedium — CHANGES_REQUESTED** [review]
Code doesn't work. Do we need to merge #53 or Develop into this one?

Additionally, while the static image added in #53 provides a good place to drop in the dynamic image, we're going to need to remove the static image that's currently there or add in a placeholder image. Can't be displaying a "rainy" image if it's not raining, you know? Even if it does get overwritten a moment later.

**Comment by jhadams82** [comment]
requested changes made, try it now

**Review by MoribundMedium — APPROVED** [review]
Legiiiiiiiit

Code is nice and clean and simple. I dig it.

**Pull request merged**